### PR TITLE
add docker image for building sigar libs

### PR DESCRIPTION
**Pull request by kovrus**

### CODE REVIEW

File: sigar/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,41 @@
+## -*- docker-image-name: "docker-crate" -*-
```
Comment (chaudum) [inline sigar/Dockerfile]: change image name

File: sigar/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+In order to support [Sigar][1] on Linux distrbutions which use [musl libc][2]
+C standard library instead of [glibc][3], e.g. [**Alpine Linux**][4]. It's
+necessary to build [**Sigar**][1] libraries from source using corresponding
+dependencies.
+
+### Build Sigar Docker Image
+
+```sh
+    docker build -t sigar .
```
Comment (chaudum) [inline sigar/README.md]: no indentation necessary

Comment (chaudum) [inline sigar/README.md]: would tag the container `crate/libsigar` - does not make any difference, though

**Comment by chaudum** [comment]
otherwise great stuff!

File: sigar/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,41 @@
+## -*- docker-image-name: "crate/libsigar" -*-
+#
+# Crate/libsigar Dockerfile
+# https://github.com/crate/docker-crate/sigar
+#
+
+FROM alpine:latest
+MAINTAINER Crate Technology GmbH <office@crate.io>
```
Comment (Nils-Magnus) [inline sigar/Dockerfile]: Should read 
MAINTAINER Crate.IO GmbH <office@crate.io>

**Comment by chaudum** [comment]
ack

**Pull request merged**